packages/mui-system/src/cssVars/createCssVarsProvider.js

```diff
@@ -126,6 +134,18 @@ export default function createCssVarsProvider(options) {
       }
     }, [colorScheme, attribute]);
 
+    React.useEffect(() => {
+      if (!mode || !enableColorScheme) {
+        return;
+      }
+      // `color-scheme` tells browser to render built-in elements according to its value: `light` or `dark`
+      if (mode === 'system') {
+        document.body.style.setProperty('color-scheme', systemMode);
+      } else {
+        document.body.style.setProperty('color-scheme', mode);
```

Comment:
From what I read:
- https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme
- https://github.com/pacocoursey/next-themes/blob/b5c2bad50de2d61ad7b52a9c5cdc801a78507d7a/index.tsx#L185

May be it is safer to use `document.documentElement` (the html tag)